resources/storage_class.py

```diff
@@ -30,22 +30,26 @@ def __init__(self, interface, **kwargs):
         self.interface = interface
         if self.interface == 'cephfs':
             self.yaml_path = os.path.join(
-                constants.TEMPLATE_CSI_RBD_DIR, "rbd_storageclass.yaml"
+                constants.TEMPLATE_CSI_FS_DIR, "storageclass.yaml"
             )
             # TODO: Implement
             pass
         if self.interface == 'rbd':
             self.yaml_path = os.path.join(
-                constants.TEMPLATE_CSI_FS_DIR, "cephfs_storageclass.yaml"
+                constants.TEMPLATE_CSI_RBD_DIR, "storageclass.yaml"
             )
             # TODO: Implement
             pass
 
         self.sc_data = yaml.safe_load(open(self.yaml_path, 'r'))
         self.sc_data.update(kwargs)
+
+        # TODO(HACK): Fix properly inheritance to OCS, storage classes don't use namespace
+        self.sc_data['metadata']['namespace'] = defaults.ROOK_CLUSTER_NAMESPACE
```

Comment:
[StorageClass](https://kubernetes.io/docs/concepts/storage/storage-classes/) objects are not in any [namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/), in the same way as other low level cluster wide resources such as nodes or PVs. If the current design doesn't allow you to specify no namespace for a storage class, the design is invalid.

Reply:
Ok, created another issue https://github.com/red-hat-storage/ocs-ci/issues/195

Reply:
> [StorageClass](https://kubernetes.io/docs/concepts/storage/storage-classes/) objects are not in any [namespace](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/), in the same way as other low level cluster wide resources such as nodes or PVs. If the current design doesn't allow you to specify no namespace for a storage class, the design is invalid.

+1.. Storageclass should not need any namespace